daemon/algod/api/algod.oas3.yml

```diff
@@ -1359,6 +1359,32 @@
   },
   "openapi": "3.0.1",
   "paths": {
+    "/genesis": {
+      "get": {
+        "description": "Returns the entire genesis file in json.",
+        "operationId": "SwaggerJSON",
```

Comment:
is that a leftover from the Swagger one?

Reply:
Ack, fixed.